TableView/Controllers/DetailController.swift

```diff
@@ -101,6 +100,19 @@ class DetailController: UIViewController {
         })
         return action
     }
+    
+    @IBAction func actionVisualize(_ sender: Any) {
+        let identifier = "Visualization"
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        guard let visualController = storyboard.instantiateViewController(withIdentifier: identifier) as? VisualizationController
+            else {
+                print("Error instantiate ViewController: \(identifier)")
+                return
+        }
+        visualController.controlManager = ControlManagerFactory.getControlManager(title: self.title!)
```

Comment:
Bad idea to rely on controller title.
Describe enumeration with types and add property of that type.
Pass property value to `ControlManagerFactory.getControlManager()`